integration_tests/pages/overview/OverviewPage.ts

```diff
@@ -9,22 +9,34 @@ export default class OverviewPage extends Page {
     super('overview')
   }
 
-  isForPrisoner(expected: string) {
+  isForPrisoner(expected: string): OverviewPage {
     this.prisonNumberLabel().should('have.text', expected)
     return this
   }
 
-  hasAddGoalButtonDisplayed() {
+  isPreInduction(): OverviewPage {
+    this.preInductionOverviewPanel().should('be.visible')
+    this.postInductionOverviewPanel().should('not.exist')
+    return this
+  }
+
+  isPostInduction(): OverviewPage {
+    this.preInductionOverviewPanel().should('not.exist')
+    this.postInductionOverviewPanel().should('be.visible')
+    return this
+  }
+
+  hasAddGoalButtonDisplayed(): OverviewPage {
     this.addGoalButton().should('be.visible')
     return this
   }
 
-  doesNotHaveAddGoalButton() {
+  doesNotHaveAddGoalButton(): OverviewPage {
```

Comment:
Added the return type of `OverviewPage` just for completeness - nothing really to do with this PR, just doing it as I was here anyway